packages/hooks/src/useWalletMigration.ts

```diff
@@ -1,8 +1,13 @@
 import { useCallback } from 'react';
 import { useFetch, useInvalidateQuery, useRequest } from '@deriv/api';
+import { useStore } from '@deriv/stores';
 
 /** A custom hook to get the status of wallet_migration API and to start/reset the migration process */
 const useWalletMigration = () => {
+    // TODO: delete it later, it's a temporary solution
```

Comment:
@sergei-deriv Could you be a bit more specific about why this temporary workaround is required?

Reply:
@nijil-deriv Is it ok if I'll write like this: 

```
// TODO: delete it later, it's a temporary solution
// because we have to check for authorize from client store before doing API call
// This hook will be refactored later for subscribe when BE is ready
```

Reply:
Is there a ticket to do this? How do we keep track of these?

Reply:
@markwylde-deriv Unfortunately I don't know and I'm not sure we have

Reply:
I've created a card under Technical debts in Trader's hub space.